UnrealPlugin/README.md

```diff
@@ -184,7 +293,7 @@ protected:
 ```
 In your MyGameInstance.h file make sure you replace the line that says class THIRDPERSONMP_API should say class [YOUR GAME NAME IN ALL CAPS_API].
```

Comment:
Remove all references to the confusing THIRDPERSONMP_API, and just use [YOUR GAME NAME IN ALL CAPS_API] or something similar in all places.